cmd/cli/kubectl-kyverno/commands/apply/command.go

```diff
@@ -298,13 +302,12 @@ func (c *ApplyCommandConfig) applyPolicytoResource(
 	return &rc, resources, skipInvalidPolicies, responses, nil
 }
 
-func (c *ApplyCommandConfig) loadResources(policies []kyvernov1.PolicyInterface, validatingAdmissionPolicies []v1alpha1.ValidatingAdmissionPolicy, dClient dclient.Interface) []*unstructured.Unstructured {
+func (c *ApplyCommandConfig) loadResources(policies []kyvernov1.PolicyInterface, validatingAdmissionPolicies []v1alpha1.ValidatingAdmissionPolicy, dClient dclient.Interface) ([]*unstructured.Unstructured, error) {
 	resources, err := common.GetResourceAccordingToResourcePath(nil, c.ResourcePaths, c.Cluster, policies, validatingAdmissionPolicies, dClient, c.Namespace, c.PolicyReport, "")
 	if err != nil {
-		fmt.Printf("Error: failed to load resources\nCause: %s\n", err)
-		osExit(1)
+		return resources, fmt.Errorf("Error: failed to load resources\nCause: %s\n", err)
 	}
-	return resources
+	return resources, nil
```

Comment:
You're not returning the error here ?

Reply:
The error is returned previously in the if statement right? Here we return the resources fetched and we received no errors, right?

Reply:
You're right, sorry